components/inputs/README.md

```diff
@@ -180,3 +181,68 @@ class MyElem extends LitElement {
 
 }
 ```
+
+## Radio Buttons
+
+Unlike checkboxes, individual radio buttons cannot be placed in a custom element. Items belonging to a radio group cannot span across different shadow roots -- all radios in the same group must be in the same shadow root.
```

Comment:
If we think there is value in wrapping, for consistency or ease of use, we _could_ provide a component that renders to the light-DOM. The node into which the template is rendered can be overridden by implementing [createRenderRoot](https://lit-element.polymer-project.org/guide/templates#renderroot)... Lit-element defaults to creating an open shadow root:
```
class InputRadio extends LitElement {
   createRenderRoot() {
      return this;
   }
}
```

Reply:
Oh _interesting_, I didn't consider that. 🤔

Reply:
I think we tried something similar to that in Polymer 3, prior to going the style module route. I think there were limitations of using light DOM which I tried to document the pros/cons of in this PR https://github.com/BrightspaceUI/inputs/pull/51.
Based on that we decided to go with a style module.
Would need to re-read that lengthy PR and absorb/compare to Lit, to determine if the problems highlighted there would still be an issue with Lit...

Reply:
Thanks for that link Martin. I tried this out last week and basically ran into all the issues you call out there -- Lit styles stopped working and no `<slot>` support. As you did, I was able to get styles working by moving them into the component itself, but for the same reasons I don't love that approach.

I think I'm leaning towards abandoning having a `<d2l-input-radio>` component and just relying on folks importing the style and applying it to native radios, which is what this PR does. At least now with Lit we have a nicer way to share these styles.